<launch>
  <arg name="cut_angle" default="0" doc="If set at [0, 360), cut at specific angle feature activated, otherwise use the fixed packets number mode."/>

  <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
    <param name="model" value="RS32"/>
    <!--param name="pcap" value="path_to_pcap"/-->
    <param name="device_ip" value="192.168.1.200"/>
    <!-- Set the Lidar data port on the PC side, default is 6699 -->
    <param name="msop_port" value="6699"/>
    <param name="cut_angle" value="$(arg cut_angle)"/>
  </node>

  <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
    <param name="model" value="RS32"/>
    <param name="curves_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/curves.csv" />
    <param name="angle_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/angle.csv" />
    <param name="channel_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/ChannelNum.csv" />
    <param name="curves_rate_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/CurveRate.csv" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz"  args="-d $(find rslidar_pointcloud)/rviz_cfg/rslidar.rviz" />

</launch>
